var/spack/repos/builtin/packages/optix/package.py

```diff
@@ -8,11 +8,11 @@
 class Optix(Package):
     """Nvidia OptiX is a ray tracing API."""
     homepage = "https://developer.nvidia.com/optix"
-    version('5.0.1','a1be032f246654ec4e4978b8c30e7774', extension='sh', expand=False)
-    url = "file:///gpfs/bbp.cscs.ch/project/proj3/development/deployment/optix.sh"
+    version('5.1.1','6e0671d49d08289c5c1ceceb9c505bbf', extension='sh', expand=False)
+    url = "file:///gpfs/bbp.cscs.ch/project/proj3/development/deployment/NVIDIA-OptiX-SDK-5.1.1-linux64-25109142.sh"
     phases = [ 'install']
```

Comment:
I am bit surprised to see the package using file from GPFS! 

Before debugging, could we replace this? See [resource](https://spack.readthedocs.io/en/latest/packaging_guide.html#resources-expanding-extra-tarballs).